plugins/tekton/README.md

```diff
@@ -162,14 +162,15 @@ Tekton is a front-end plugin that enables you to view the `PipelineRun` resource
 
 1. Go to the **CI/CD** tab.
 
-   The **CI/CD** tab displays the latest `PipelineRun` resources associated to a Kubernetes cluster. The resources include tasks to complete. When you hover the mouse pointer on a task card, you can view the steps to complete that particular task.
+   The **CI/CD** tab displays the latest `PipelineRun` resources associated with a Kubernetes cluster. The resources include tasks to complete. When you hover the mouse pointer on a task card, you can view the steps to complete that particular task.
 
    ![ci-cd-tab-tekton](./images/tekton-plugin-user1.png)
 
    There is also a **GO TO TEKTON** option at the bottom, which redirects you to the **TEKTON** tab.
 
-1. Click **GO TO TEKTON** or select the **TEKTON** tab in the entity view page.
+1. Click **GO TO TEKTON** or select the **TEKTON** tab on the entity view page.
 
    The **TEKTON** tab contains the list of pipeline runs related to a cluster. The list contains pipeline run details, such as **NAME**, **STATUS**, **TASK STATUS**, **STARTED**, and **DURATION**.
+   
+   ![usertekton-tab](./images/tekton-plugin-user2.png)
 
```

Comment:
Please remove the empty line at the end of the file to make it pass the CI,

or run `yarn prettier:fix` locally to fix it automatically.

Reply:
I did run the command, but it's showing the following error. Not sure, if I'm doing something wrong:

heenamanwani@hmanwani-mac backstage-plugins % yarn prettier:fix
yarn run v1.22.19
$ prettier --write .
[error] Invalid configuration for file "/Users/heenamanwani/Documents/repos/backstage-plugins/.eslintrc.js":
[error] Cannot find module '@spotify/prettier-config'
[error] Require stack:
[error] - /Users/heenamanwani/Documents/repos/backstage-plugins/.prettierrc.js
[error] - /usr/local/lib/node_modules/prettier/internal/internal.mjs
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.